test/DurableTask.SqlServer.AzureFunctions.Tests/IntegrationTestBase.cs

```diff
@@ -152,11 +164,12 @@ protected async Task<DurableOrchestrationStatus> RewindOrchestrationAsync(string
             return status;
         }
 
-        protected async Task<IDurableClient> GetDurableClientAsync()
+        protected IDurableClient GetDurableClient(string? taskHubName = null)
         {
-            var clientRef = new IDurableClient[1];
-            await this.CallFunctionAsync(nameof(ClientFunctions.GetDurableClient), "clientRef", clientRef);
-            IDurableClient client = clientRef[0];
+            var client = this.clientFactory.CreateClient(new DurableClientOptions
```

Comment:
Nice to see that you found a way around my ugly hack for getting a client. For some reason I never considered using the client factory. 😅